tapfreighter/wallet.go

```diff
@@ -554,8 +570,13 @@ func (f *AssetWallet) fundPacketWithInputs(ctx context.Context,
 	fundDesc *tapsend.FundingDescriptor, vPkt *tappsbt.VPacket,
 	selectedCommitments []*AnchoredCommitment) (*FundedVPacket, error) {
 
+	assetId := fundDesc.AssetSpecifier.UnwrapIdToPtr()
```

Comment:
Could also add an `UnwrapIdOrErr()` method, so we can return a named error constant for `unable to unwrap asset ID`.